picoquic/packet.c

```diff
@@ -109,7 +125,7 @@ int picoquic_parse_packet_header(
                             ph->ptype = picoquic_packet_server_stateless;
                             break;
                         case 0xFD:
-                            ph->ptype = (to_server == 0) ? picoquic_packet_server_cleartext : picoquic_packet_client_cleartext;
+                            ph->ptype = is_client ? picoquic_packet_server_cleartext : picoquic_packet_client_cleartext;
                             break;
```

Comment:
The two types "server clear text" and "client clear text" were defined in the early drafts. They have been unified in a single "handshake" type in draft 08. We might be able to simplify the code by just handling a single packet type instead on insisting on 2. I opened issue #63 for that.

Reply:
Ahh that is nice :)